gap/examples.gi

```diff
@@ -369,6 +369,134 @@ GeneralisedPetersenGraphCons);
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
 
+InstallMethod(BishopsGraphCons,
+"for IsMutableDigraph, a string and two positive integers",
+[IsMutableDigraph, IsString, IsPosInt, IsPosInt],
+function(filt, color, m, n)
+  local D, D1, D2, i, j, v;
+
+  if not (color = "dark-square" or color = "light-square") then
+    ErrorNoReturn(
+    "the argument <color> must be either \"dark-square\" or \"light-square\".");
+  fi;
+
+  D1 := EmptyDigraph(filt, m * n);
+  D2 := EmptyDigraph(filt, m * n);
+
+  for i in [1 .. (m - 1)] do
+    for j in [1 .. (n - 1)] do
+      if IsEvenInt(i + j) and color = "dark-square"
+          or IsOddInt(i + j) and color = "light-square" then
+        v := (i - 1) * n + j;
+        DigraphAddEdge(D1, [v, v + n + 1]);
+      fi;
+    od;
+  od;
+  for i in [1 .. (m - 1)] do
+    for j in [1 .. (n - 1)] do
+      if IsEvenInt(i + j) and color = "light-square"
+          or IsOddInt(i + j) and color = "dark-square" then
+        v := (i - 1) * n + j + 1;
+        DigraphAddEdge(D2, [v, v + n - 1]);
+      fi;
+    od;
+  od;
```

Comment:
```suggestion
   for i in [1 .. (m - 1)] do
     for j in [1 .. (n - 1)] do
       if IsEvenInt(i + j) and color = "dark-square"
           or IsOddInt(i + j) and color = "light-square" then
         v := (i - 1) * n + j;
         DigraphAddEdge(D1, [v, v + n + 1]);
       elif IsEvenInt(i + j) and color = "light-square"
           or IsOddInt(i + j) and color = "dark-square" then
         v := (i - 1) * n + j + 1;
         DigraphAddEdge(D2, [v, v + n - 1]);
       fi;
     od;
   od;
```
I feel that this would make it neater. It works on my computer when I tried it :)

Reply:
Yes! That's a good point, the way I wrote it was really redundant there. Thanks for pointing that out.